app/src/main/java/org/mozilla/fenix/settings/SettingsFragment.kt

```diff
@@ -136,11 +139,21 @@ class SettingsFragment : PreferenceFragmentCompat() {
     }
 
     override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
-        val preferencesId = if (FeatureFlags.newIconSet) {
-            R.xml.preferences_without_icons
+
+        val preferencesId = if (isExperimentBranch()) {
+            if (FeatureFlags.newIconSet) {
+                R.xml.preferences_without_icons_experiment
+            } else {
+                R.xml.preferences_experiment
```

Comment:
As this `XML` belongs to a specific experiment may we should add the name of the experiment to identify it